lib/action_cable/test_helper.rb

```diff
@@ -45,6 +45,13 @@ def after_teardown # :nodoc:
     #   end
     #
     def assert_broadcasts(target, number, channel: nil)
+      unless channel.nil?
```

Comment:
Alas, we can't refuse channel because `broadcasting_for`  requires a channel name to build a stream in a base of the channel name. Current AC allows  to use `Channel.broadcast_to(obj, message)` in any place of your code